### PR TITLE
fix: use the context correctly in mt_bench_branch

### PR DESCRIPTION
**Pull request by bcrochet**
The key is supposed to be context, not question.

### CODE REVIEW

**Review by alimaredia — APPROVED** [review]

File: src/instructlab/eval/mt_bench_branch_generator.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def generate(judge_model_name, branch, taxonomy_dir, output_dir):
                 if q is None or a is None:
                     continue
 
-                c = ex["question"] if "context" in ex else None
+                c = ex["context"] if "context" in ex else None
```
Comment (booxter) [inline src/instructlab/eval/mt_bench_branch_generator.py:63]: nit: This would be the same as `c = ex.get("context")` I think.

Comment (danmcp) [reply src/instructlab/eval/mt_bench_branch_generator.py:63]: +1

Comment (danmcp) [reply src/instructlab/eval/mt_bench_branch_generator.py:63]: I think this is worth a quick change to avoid any future confusion as well.

Comment (bcrochet) [reply src/instructlab/eval/mt_bench_branch_generator.py]: Done.

**Review by nathan-weinberg — APPROVED** [review]
I will merge once the E2E run completes

**Pull request merged**